docs/en/administration/workdir_caching.md

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
Don't we need the documentation in German too?

Reply:
I thought this is only mandatory for user documentation (in path `docs/en/user`).?.